### ISSUE TITLE
BaseUlt, RandomUlt, Snitched and other utilities

### ISSUE THREAD

**Issue by serseriler22** [berbb/PortAIO-v2#76]
Can you add this addons ?

**Comment by Lemonadez** [comment]
No.. He is not looking for a Request. But a bug list of all champion. Utility n all that crap r not in priority.

**Comment by BigPhalus** [comment]
dont know what the guy above me is talking about.... but anyway yea all these addons will be added soon, berbb is adding all L# utilities soonish

**Comment by Lemonadez** [comment]
https://github.com/berbb/PortAIO-v2/issues/57

**Comment by BigPhalus** [comment]
Doctor Berb MD - 10/29/2016
@everyone http://prntscr.com/d0j6ku - "Is X going to be ported?" If X is a utility then YES it will be ported.(edited)

https://discordapp.com/channels/177241543600635904/239145255679688705

**Comment by Lemonadez** [comment]
That was was on 10-29 (Oct 29)

This was from 10 Days ago. It was stated he is not taking a request.
https://github.com/berbb/PortAIO-v2/issues/57

Maybe ur not reading the date?
